pkg/spdk/backing_image.go

```
@@ -0,0 +1,703 @@
+	}()
+	if bi.State == types.BackingImageStateReady {
+		updateRequired = false
+		return nil, grpcstatus.Errorf(grpccodes.AlreadyExists, "backing image %v already exists and running", bi.Name)
+	}
+	if bi.State != types.BackingImageStateStarting {
+		updateRequired = false
+		return nil, fmt.Errorf("invalid state %s for backing image %s creation", bi.State, bi.Name)
+	}
+
+	go func() {
+		bi.prepareBackingImageSnapshot(spdkClient, superiorPortAllocator, fromAddress, srcLvsUUID)
```

golangci-lint (1.62.2)

Error return value of `bi.prepareBackingImageSnapshot` is not checked

(errcheck)